### PR TITLE
IA-2992: View only permission for Planning

### PR DESCRIPTION
**Pull request by hakifran**
Explain what problem this PR is resolving
- It would be useful to allow some users to visualize plannings without having any right to modify those plannings.
Related JIRA tickets : [IA-2992](https://bluesquare.atlassian.net/browse/IA-2992)

## Self proofreading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [x] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes
- Add iaso_planning_read and iaso_planning_write permissions
- create migration to link user with iaso_registry and iaso_planning with the new permissions
- Display Edit, delete, copy, create when the user has iaso_planning_write

## How to test
- Edit or create a user and remove the iaso_planning_write permission
- check if the edit, delete, duplicate and create buttons are there anymore
## Print screen / video
[Screencast from 2024-05-30 18-12-43.webm](https://github.com/BLSQ/iaso/assets/19631540/d55899b3-6cdd-41f7-96a0-ee6929442c18)

[IA-2992]: https://bluesquare.atlassian.net/browse/IA-2992?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by beygorghor — APPROVED** [review]
Maybe ask a python guy to have a look on the migration to be sure i'm not missing something

File: hat/menupermissions/models.py [hunk 1]
```diff
@@ -239,7 +241,8 @@ class Meta:
             (_POLIO_NOTIFICATIONS, _("Polio notifications")),
             (_SUBMISSIONS, _("Soumissions")),
             (_SUBMISSIONS_UPDATE, _("Editer soumissions")),
-            (_PLANNING, _("Planning")),
+            (_PLANNING_WRITE, _("Editer le planning")),
+            (_PLANNING_READ, _("lire le planning")),
```
Comment (kemar) [inline hat/menupermissions/models.py]:
```suggestion
            (_PLANNING_READ, _("Lire le planning")),
```

File: hat/menupermissions/migrations/0061_alter_custompermissionsupport_options.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                    ("iaso_projects", "Projets"),
+                    ("iaso_sources", "Sources"),
+                    ("iaso_data_tasks", "Tâches"),
+                    ("iaso_polio", "Polio"),
+                    ("iaso_polio_config", "Polio config"),
+                    ("iaso_polio_notifications", "Polio notifications"),
+                    ("iaso_submissions", "Soumissions"),
+                    ("iaso_update_submission", "Editer soumissions"),
+                    ("iaso_planning_write", "Editer le planning"),
+                    ("iaso_planning_read", "lire le planning"),
```
Comment (kemar) [inline hat/menupermissions/migrations/0061_alter_custompermissionsupport_options.py]:
```suggestion
                    ("iaso_planning_read", "Lire le planning"),
```

**Review by kemar — APPROVED** [review]
LGTM

**Pull request merged**